core/src/main/java/com/yandex/yoctodb/mutable/DocumentBuilder.java

```diff
@@ -32,7 +32,8 @@
 
     enum LengthOption {
         FIXED,
-        VARIABLE
+        VARIABLE,
+        TRIE
```

Comment:
It is quite unexpected to see this `enum` being extended...

Reply:
Yep, it feels not very natural. Looks like we need to introduce another way to control index serialization on a time index being created, what do you think?

Reply:
Let's rename the `enum` to `IndexType` and `FIXED`/`VARIABLE` to something more appropriate.